docs/_conversion/maintaining.md

```diff
@@ -0,0 +1,140 @@
+
+You can also add 'subscribers'.  Those folks will get emails about the integration's performance, but won't be allowed to edit the integration. 
+
+![](https://cdn.zappy.app/a3f77d24190e718b6177efe3d7074736.png)
+
+## Sharing
+
+Have a set of users who will help you test your new integration versions prior to making them publicly available?  Go to the "Sharing" section of the UI and invite those users to give them access to versions of integration other than the public, "promoted" version. 
+
+## Analytics
+
+Check out how many Zaps are using each trigger & action, for each version of your integration.  We'll show you how many are enabled vs how many are paused.  
+
+![](https://cdn.zappy.app/67ad08f295dde5fc914dd74fb7819c67.png)
+
+We're looking to bring out lots more data and insights about your integrations performance and usage dynamics in the near future.
+
+## Where’s my code?
+
+Your converted project looks a little different than if you’d started it from scratch in the new platform, but should feel pretty familiar (apart from versioning).  One thing you may notice are sections where the configuration is a code block, and the code is simply calling `z.legacyScripting.run()`.  To make integrations written in the old environment run, without modification, in the new environment we 'wrapped' your custom code and API configuration, and created a runtime library that emulates the behavior of the old platform.  So each time you see this, you can recognize it as an artifact of this emulation - we're calling into the emulator to perform the operation we need using your original code and configuration.
+
+Where you added custom code to override lifecyle methods in Web Builder, we've included that code in the "Advanced" section of the UI, in the "Legacy Web Builder" tab.  The `z.legacyScripting` will call these functions to drive your triggers, actions, and authentication transactions.  
+
+![](https://cdn.zappy.app/79bb2a1ea31a301f286e07e209994dcd.png)
+
+
+
+
+## Creating new triggers and actions
+
+You'll be able to continue your development, creating new triggers and actions, in your converted Web Builder integration.  When you create a new trigger or action, the interface will look a little bit different than those triggers and actions that were converted over from the legacy platform.  It will look familiar to you.  The big difference is when configuring an HTTP API request, you'll have many more options to customize headers and parameters without having to write custom code.  And when you do need to use custom code, you'll do so in that trigger/actions's UI, rather than in a common script file.  We'll talk about differences in the scripting environment next.
```

Comment:
```suggestion
You'll be able to continue your development, creating new triggers and actions, in your converted Web Builder integration.  When you create a new trigger or action, the interface will look a little bit different than those triggers and actions that were converted over from the legacy platform.  It will look familiar to you.  The big difference is when configuring an HTTP API request, you'll have many more options to customize headers and parameters without having to write custom code.  And when you do need to use custom code, you'll do so in that trigger/actions' UI, rather than in a common script file.  We'll talk about differences in the scripting environment next.
```